.github/workflows/agw-workflow.yml

```diff
@@ -0,0 +1,301 @@
+            sudo apt-get install -y python3-aioeventlet
+            sudo rm -rf /var/lib/apt/lists/*
+      - name: Install libraries and dependecies
+        run: |
+            mkdir -p /var/tmp/test_results
+            mkdir -p /var/tmp/codecovs
+            sudo apt-get update -y
+            sudo apt-get install -y libsystemd-dev pkg-config curl zip unzip
+            sudo apt-get install -y virtualenv python-babel python-dev build-essential python3-setuptools python-setuptools autogen autoconf libtool python3-apt python3-requests python3-pip python-protobuf
+            pip3 install setuptools==49.6.0
+      - name: Setup protoc3
+        run: |
+            sudo curl -Lfs https://github.com/google/protobuf/releases/download/v3.1.0/protoc-3.1.0-linux-x86_64.zip -o protoc3.zip
+            sudo unzip protoc3.zip -d protoc3
+            sudo mv protoc3/bin/protoc /bin/protoc
+            sudo chmod a+rx /bin/protoc
+            sudo mv protoc3/include/google /usr/include/
+            sudo chmod -R a+Xr /usr/include/google
+            sudo rm -rf protoc3.zip protoc3
+      - name: Setup swagger codegen
+        env:
+          CODEGEN_ROOT: "${{ env.MAGMA_ROOT }}/.codegen"
+          SWAGGER_CODEGEN_JAR: "${{ env.MAGMA_ROOT }}/.codegen/swagger-codegen-cli.jar"
+        run: |
+            sudo -E mkdir ${{ env.CODEGEN_ROOT }}
+            sudo -E wget https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.2.3/swagger-codegen-cli-2.2.3.jar -O ${{ env.SWAGGER_CODEGEN_JAR }}
+            sudo -E chmod -R 755 ${{ env.CODEGEN_ROOT }}
+      - name: run test_all
+        env:
+          PYTHON_BUILD: "${{ env.MAGMA_ROOT }}/build"
+          PIP_CACHE_HOME: "${{ env.MAGMA_ROOT }}/.pipcache"
+          SWAGGER_CODEGEN_JAR: "${{ env.MAGMA_ROOT }}/.codegen/swagger-codegen-cli.jar"
+          MAGMA_DEV_MODE: 1
+          SKIP_SUDO_TESTS: 1
+        run: |
+            sudo -E make -C ${{ env.MAGMA_ROOT }}/lte/gateway/python test_all
+      - name: Upload Test Results
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: Unit Test Results
+          path: /var/tmp/test_results
+      - uses: codecov/codecov-action@v1
+        with:
+          files: '/var/tmp/codecovs/cover_lte.xml'
+          flags: lte-test
+      - uses: codecov/codecov-action@v1
+        with:
+          files: '/var/tmp/codecovs/cover_orc8r.xml'
+          flags: lte-test
+      # Notify ci channel when failing
+      # Plugin info: https://github.com/marketplace/actions/slack-notify
+      - name: Notify failure to slack
+        if: failure() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "Github action lte-test update failed"
+          SLACK_USERNAME: "AGW workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+  session_manager_test:
```

Comment:
let's just add a space to separates the jobs in a same workflow. just for readability

Reply:
@quentinDERORY  I believe we used to have a new line before a new job but we decided to remove it. 
Shall we follow @tmdzk suggestion and add them back?

Reply:
let's do it :)

Reply:
done :)